index.html

```diff
@@ -2891,6 +2891,40 @@ <h3 id="applying">
         </ol>
       </section>
     </section>
+      <section>
+        <h3>
+          <code title="">iarc_rating_id</code> member
+        </h3>
+        <p>
+          The <dfn id="member-iarc_rating_id"><code>iarc_rating_id</code> member</dfn>
+          is a <a>string</a> that represents an ID value of the IARC rating of the 
```

Comment:
That would be great, but when I looked I am not sure these things are actually public, so maybe we could create some examples in the spec instead or some guidance? ie a Note?